helix_fhir_client_sdk/fhir_client.py

```diff
@@ -1620,6 +1623,21 @@ async def refresh_token(
 
         return refresh_token
 
+    @staticmethod
+    def get_variables_to_log(vars_dict: Dict[str, Any]) -> Dict[str, Any]:
+        """
+        Method to return the variables which we need to log
+        :param vars_dict: (dict) dictionary of variables names with their values
+        """
+        variables_to_log = {}
+        for key, value in vars_dict.items():
+            if not isinstance(value, type(threading.Lock)):
```

Comment:
Is there any case where this type of value is coming? Please mention here.

Reply:
there are some variables which contains objects. On these objects we are having lock due to which we were not able to add those into our variables dict which we need to log.